cedar-policy-validator/src/cedar_schema/test.rs

```diff
@@ -1859,6 +1862,57 @@ mod translator_tests {
         .map(|_| ());
         assert_matches!(schema, Err(_));
     }
+
+    #[test]
+    fn reserved_json_schema_keyword() {
+        let schema = cedar_schema_to_json_schema(
+            parse_schema(
+                r#"
+               type Entity = Long;
+        "#,
+            )
```

Comment:
Can we also get tests for `namespace N { type Entity = Long; }` and similar?

Reply:
Done.